src/Sarif.Multitool/PageOptions.cs

```diff
@@ -50,7 +50,7 @@ internal class PageOptions
             "force",
             Default = true,
             HelpText = "Force overwrite of output file if it exists.")]
-        public bool Force { get; set; } = true;
```

Comment:
There is a bug #1630, "Multitool page command --force defaults to true and can't be changed". This change does not fix that bug!

Rather, this change makes it easier to write my new unit tests in PageCommandTests. Without this change, I had to explicitly set `Force = false` in every one of my test cases except for the single case where `Force` needed to be `true`.

It makes sense to remove this initialization; I don't think there's a single other property in any of the options classes that's initialized like this. It also makes sense to fix #1630 by removing the `Default = true` on line 51, but I don't want to do that in this PR because it's a command line behavior change and I want to validate it with you first.

Note that with `Default = true`, not only do I think it's the wrong default, but there's no way to specify the value "false" on the command line! There's nothing like PowerShell's `-mySwitch:false` syntax to turn it off.